pkg/imrpc/instance.proto

```diff
@@ -0,0 +1,92 @@
+syntax="proto3";
+
+package imrpc;
+
+import "google/protobuf/empty.proto";
+import "imcommon.proto";
+
+service InstanceService {
+    rpc InstanceCreate(InstanceCreateRequest) returns (InstanceResponse) {}
+	rpc InstanceDelete(InstanceDeleteRequest) returns (InstanceResponse) {}
```

Comment:
Make sure the code formatted. It seems wrong indent here.

Reply:
Weird VSCode... It said it is formatted.

Reply:
Addressed. Thank you.